data_structures/binary_trees/words_alphabetical.c

```diff
@@ -0,0 +1,226 @@
+/**
+ * @file
+ * @brief Printing the words contained in a file "file.txt" in alphabetical
+ * order and also their frequencies in to another file "wordcount.txt"
+ */
+#include <ctype.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
```

Comment:
Please add a one-line description of what the library/header is for (**see the example below**).

```c
#include <stdio.h>    /// for IO operations
#include <assert.h>  /// for assert
```

Reply:
Comments has been added